react/features/base/conference/reducer.js

```diff
@@ -47,6 +48,7 @@ ReducerRegistry.register(
             return _authStatusChanged(state, action);
 
         case CONFERENCE_FAILED:
+        case CONFERENCE_RESTARTED:
```

Comment:
Not relevant anymore after the refactor but when enableForcedReload is false, none of this gets triggered in the client since it won't receive a CONFERENCE_RESTARTED error from lib-jitsi-meet.

Reply:
Yep I saw this. But then there comes the question why do we check for the config property at all then. It just felt a little inconsistent...

Anyway  as you mentioned, not relevant anymore!